### DISCUSSION TITLE
Serverless EMR Support

### DISCUSSION THREAD

**Discussion by AmitArie** [dagster-io/dagster#11459] [Ideas]
Hello my fellow Dagsters,
Is there a plan to support Serverless EMR instead of the regular one?
Thanks!

**Comment by sryza** [comment]
Hi @AmitArie - I could definitely see the value in this, but it's not something on our near-term roadmap. If you'd to submit a PR, we'd be happy to take a look. Btw, if you file a github issue for this, it will help us track it.

**Comment by kdeelaka-superloop** [comment]
This seems like it's coming soon: https://github.com/dagster-io/dagster/pull/24318
